react/features/base/redux/functions.js

```diff
@@ -25,6 +26,19 @@ export function assign(target: Object, source: Object) {
     return t;
 }
 
+/**
+ * Wrapper function for the react-redux connect function to avoid having to
+ * declare function types for flow, but still let flow warn for other errors.
+ *
+ * @param {Function} mapStateToProps - Redux mapStateToProps function.
+ * @param {Function?} mapDispatchToProps - Redux mapDispatchToProps function.
+ * @returns {Connector}
+ */
+export function connect(
+        mapStateToProps: Function, mapDispatchToProps?: Function) {
+    return reduxConnect<*, *, *, *, *, *>(mapStateToProps, mapDispatchToProps);
```

Comment:
Can you try to use the new type placeholder `_` as stated in https://github.com/jitsi/jitsi-meet/blob/master/flow-typed/npm/react-redux_v5.x.x.js ?

I think The Right Way (TM) to do this may require a type definition on each usage, so the Props and OwnProps are typechecked, but I'm not sure.

Reply:
I trued, but I get this: ```Please use a concrete type annotation instead of `_` in this position.``` :(

Reply:
The best I could come up with was this: ```<*, *, _, _, _, *>``` but then what's the point of the new placeholder? 🤔

Reply:
Ok, let's leave it then.